apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/utils/contains-missing-block.js

```diff
@@ -0,0 +1,28 @@
+// Once parsed, missing Blocks have a name prop of `core/missing`.
+// see: https://github.com/WordPress/gutenberg/tree/742dbf2ef0e37481a3c14c29f3688aa0cd3cf887/packages/block-library/src/missing
+const MISSING_BLOCK_NAME = 'core/missing';
+
+/**
+ * Determines whether the provided collection of Blocks contains any "missing"
+ * blocks as determined by the presence of the `core/missing` block type.
+ *
+ * @param {Array} blocks the collection of block objects to check for "missing" block .
+ * @returns {boolean} whether the collection blocks contains any missing blocks.
+ */
+function containsMissingBlock( blocks ) {
```

Comment:
minor: does it make sense returning immediately if the blocks array is empty?

```es6
if ( ! blocks || ! blocks.length ) {
  return [];
}
```

Reply:
I see why you are asking. However, the purpose of the function is to detect whether the blocks "contains the missing block". An empty set of blocks _does not_ contain the missing block. Therefore IMHO it is valid.

Do you have a particular concern about it I might have missed?

Also, note that the blank template doesn't have any blocks so if we didn't account for this then the blank template would never show. However, note that this early return is not directly linked to the blank template - it's just a nice side effect.

Reply:
I just was testing this code and saw that the blocks array was empty. So in order to avoid calling the `find()` function, return `false` immediately. That was my point. However, I prelabeled with `minor`, so feel free to ignore it :-D

Reply:
Oh I see! This code isn't there and this is a suggestion. Whoops! Yes we should add it. My bad!